.circleci/config.yml

```diff
@@ -227,45 +227,45 @@ jobs:
           path: ./dist
           destination: packages
 
-
 workflows:
   version: 2
   test_and_build:
     jobs:
       - check
       - build-linux:
           requires:
-             - check
+            - check
       - build-win-no-installer:
           requires:
-             - check
+            - check
           filters:
             branches:
               ignore:
-                - /^(v|V)\.\d+\.\d+(\.\d+)?(-(rc|RC)-.*/)?/
+                - /^release-\d+\.\d+(\.\d+)?(-(rc|RC).*/)?/
```

Comment:
Looks like there is an extraneous '/' at the end of the last part: '(-(rc|RC).*/)'?

Also, do we want uppercase 'RC' as an option? 0/5